src/logic/operate.js

```diff
@@ -23,7 +23,7 @@ export default function operate(numberOne, numberTwo, operation) {
     try {
       return one.mod(two).toString();
     } catch (err) {
-      return "Can't find modulo as can't divide by 0.";
+      return "Can't find module as can't divide by 0.";
```

Comment:
- [ ] I noticed you make some changes here but the `%` button is named `modulo`,  in definition the `modulo` is the remainder after dividing one number by another, and `module` is one of the separate units that a course of study has been divided into.